### PR TITLE
Polish readme

### PR DESCRIPTION
**Pull request by jcace**
Including a bugfix for the DB configuration code and a Makefile

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,11 +1,88 @@
...
-## Generating Swagger
+The Tenant services handle all aspects of Spade Storage Proposals from the Tenant side. At a high level, this includes interfaces to manage the following:
+- Policies
+- Storage Contracts
+- Collections
+- Content (i.e, sources of Filecoin Pieces)
+- Replication Constraints
+- Storage Provider SLA Compliance
+
+A high level architecture diagram of the Tenant functionalities and their relationships can be found [here](https://miro.com/app/board/uXjVMwfo8TM=/?share_link_id=267050417898)
```
Comment (stephen-pl) [inline readme.md:18]: This is marked as internal in miro. Should it be shared?

File: readme.md [hunk 2]
```diff
@@ -1,11 +1,88 @@
...
+- Content (i.e, sources of Filecoin Pieces)
+- Replication Constraints
+- Storage Provider SLA Compliance
+
+A high level architecture diagram of the Tenant functionalities and their relationships can be found [here](https://miro.com/app/board/uXjVMwfo8TM=/?share_link_id=267050417898) 
+
+<img src="docs/arch-diagram.png" width=500 />
+
+# API
+All functionality is exposed over a REST API. Live API docs can be viewed at https://spade.storage (Coming soon), or inside the repository [docs folder.](https://github.com/data-preservation-programs/spade-tenant/tree/main/docs)
```
Comment (stephen-pl) [inline readme.md:23]: Should we include this with everything that is going on

**Comment by stephen-pl** [comment]
Overall looks good to me. Left a few comments.

File: readme.md (hunk 2 above)
Comment (jcace) [reply readme.md:23]: good point - will reword to make it more open-ended

File: readme.md (hunk 1 above)
Comment (jcace) [reply readme.md:18]: It's publically viewable with the link, should be okay.

**Review by stephen-pl — APPROVED** [review]

**Comment by stephen-pl** [comment]
LGTM :rocket:

**Pull request merged**